manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Gitpod Online IDE",
   "short_name": "Gitpod",
   "version": "1.2",
-  "description": "Gitpod - One-Click Online IDE for GitHub, GitLab and Bitbucket",
+  "description": "Gitpod – One-Click Online IDE for GitHub and GitLab",
```

Comment:
I would argue that, if we request `bitbucket.org` permission, we might as well mention `Bitbucket` in the description. Sure, gitpod.io doesn't support bitbucket.org today, but I would argue that it's a One-Click Online IDE intended for GitHub, GitLab and Bitbucket.

Reply:
Also, removing it makes it look like we're backpedaling and ended up deciding that we're not going to support Bitbucket.